theano/tensor/tests/test_basic.py

```diff
@@ -1824,14 +1826,43 @@ def test_py_c_match():
     inplace=True,
     skip=skip_scipy)
 
+_good_broadcast_binary_polygamma = dict(
+    normal=(randint_ranged(0, 3, (2, 3)), rand_ranged(1, 10, (2, 3)),),
```

Comment:
Can the first parameter be a tensor? The error in travis make me think it need to be a scalar.